### PR TITLE
fix: the responsive display of blog and writing blog page

### PR DESCRIPTION
**Pull request by abhishekjakhar**
### DESCRIPTION

The Blog page and Writing blog page are having extra 20px padding on left and right side each, which is making the display very narrow on mobile phones. This is happening because the Blog and Writing blog page have been given maximum width of 90vw, which limits it from expanding to the full width.

The blog and writing blog page are being passed down to the markdown-page.js template which is wrapping all of it's children with `<Container/> `and the default maximum width value of `<Container/>` is 800px which is being used other pages also for example about, links, info etc.

The 800px is same as 90vw, so removing the 90vw will not affect in any way, instead it will make the layout consistent(blog and writing blog page will have the same maximum width as about, links, info, appearances etc pages)

#### Expected Behaviour

blog-fixed

writing-blog-fixed

#### Actual Behaviour

blog-bug

writing-blog-bug

### HOW TO TEST

- Go to blogs page(http://localhost:8000/blog/) and Writing Blog page(http://localhost:8000/writing/blog/)
- The page should not be having extra 20px padding on left and right side, which means it should align with starting of header logo and end of hamburger menu.

### CODE REVIEW

**Comment by kentcdodds** [comment]
Thanks for this! I think we'll need a different solution, because doing this on desktop prevents it from being the full `90vw` which I need for these pages.

**Comment by abhishekjakhar** [comment]
> Thanks for this! I think we'll need a different solution, because doing this on desktop prevents it from being the full `90vw` which I need for these pages.

Should I make all the pages inside markdown-page.js template from pixels to relative VW unit, will that work?

**Comment by abhishekjakhar** [comment]
> Thanks for this! I think we'll need a different solution, because doing this on desktop prevents it from being the full `90vw` which I need for these pages.

Right now, only blog and writing-blog are using vw unit, other pages which are getting passed inside markdown-page template as children are still in pixel units

**Comment by kentcdodds** [comment]
What if we removed the left and right padding?

**Comment by abhishekjakhar** [comment]
> What if we removed the left and right padding?

I have updated the code, this time I have only removed left and right padding

File: src/pages/writing/blog.mdx [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Writing Blog | Kent C. Dodds
 description: Search Kent C. Dodds writing blog posts
 maxWidth: '90vw'
+noMobileHorizontalPadding: true
```
Comment (kentcdodds) [inline src/pages/writing/blog.mdx:5]: I'd really love to not have to specify this... Can't we just remove the horizontal padding for everything?

Comment (abhishekjakhar) [reply src/pages/writing/blog.mdx:5]: @kentcdodds Removing horizontal padding for everything will also remove the padding for other pages which are getting passed down markdown-page template for example about, links, info etc.
I can check the route inside markdown-page template and based on that I can remove the padding for only /blog and /writing/blog/ on smaller screen sizes?

Comment (abhishekjakhar) [reply src/pages/writing/blog.mdx:5]: @kentcdodds We can also use maxWidth: '90vw' for all the other pages like we did in /blog and /writing/blog, that will also fix the problem?

**Comment by kentcdodds** [comment]
Thanks!

**Pull request merged**